config/gdi-starter-kit/docker-compose.override.yml

```diff
@@ -138,6 +138,13 @@ services:
     networks:
       - secure
 
+  funnel:
+    networks:
+      - secure
+      - public
+    ports:
+      - 8111:8000
```

Comment:
Might be convenient to make the external port configurable through an environment variable.